browser/src/ContactPage.tsx

```diff
@@ -19,14 +19,14 @@ export default () => (
 
     <p>
       For questions about the OurDNA dataset, please see our blog.{' '}
-      <Link href="/news"></Link>.
+      <Link href="/news"/>.
     </p>
 
     <p>
       Follow us on Instagram{' '}
       <ExternalLink href="https://www.instagram.com/ourdna_australia/">@ourdna_australia</ExternalLink>.
      and Facebook{' '}
```

Comment:
I guess you want to link this to FB page, can be empty text between start and end tag.